src/act.other.cpp

```diff
@@ -1509,7 +1509,8 @@ ACMD(do_skills)
         continue;
 
       if (GET_POWER_TOTAL(ch, i) > 0) {
-        snprintf(buf2, sizeof(buf2), "%-20s", adept_powers[i]);
+        snprintf(ENDOF(buf), sizeof(buf), "\r\nPP      Ability              Level\r\n")
```

Comment:
Did you test this? Using `ENDOF(buf)` here without initializing the buf means you're gonna have random junk at the beginning of your abilities command. You'll want to use standard `buf` instead.

Reply:
Also, you put this inside the for-loop, it's gonna stick itself in buf for every power you have active.